CMake/sign-exe.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
```

Comment:
I don't think we should be shipping this file publicly

Reply:
Already gone in the latest commit. Will need to find a way to patch the checkout in Arm CI though - I do want the ability to sign the GitHub releases.

Reply:
I think you can probably build the signing tools into some docker image or install them on the build node for windows and have jenkins invoke them directly